components/assetstore-controller-manager/Makefile

```diff
@@ -1,9 +1,18 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+APP_NAME ?= assetstore-controller-manager
+IMG ?= $(APP_NAME):latest
+IMG-CI = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME):$(DOCKER_TAG)
 
 all: test manager
 
+# Resolve dependencies
+resolve:
+	dep ensure -v -vendor-only
+
+setup:
+	go get github.com/vektra/mockery/.../
+
```

Comment:
We can also modify `deploy` target - run `install` in `deploy` target, to avoid two lines of duplicated code 😛